generator/src/main/java/com/faforever/neroxis/generator/cli/BasicOptions.java

```diff
@@ -5,20 +5,22 @@
 import lombok.Setter;
 import picocli.CommandLine;
 
+import java.util.Random;
+
 @Getter
 @Setter
 public class BasicOptions {
     @CommandLine.Spec
     CommandLine.Model.CommandSpec spec;
-    @CommandLine.Option(names = "--seed", order = 1, description = "Seed for the generated map")
-    private Long seed;
-    @CommandLine.Option(names = "--spawn-count", order = 3, defaultValue = "6", description = "Spawn count for the generated map", showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
+    @CommandLine.Option(names = "--seed", order = 3, description = "Seed for the generated map")
+    private Long seed = new Random().nextLong();
+    @CommandLine.Option(names = "--spawn-count", order = 5, defaultValue = "6", description = "Spawn count for the generated map", showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
     private Integer spawnCount;
-    @CommandLine.Option(names = "--num-teams", order = 4, defaultValue = "2", description = "Number of teams for the generated map (0 is no teams asymmetric)", showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
+    @CommandLine.Option(names = "--num-teams", order = 6, defaultValue = "2", description = "Number of teams for the generated map (0 is no teams asymmetric)", showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
     private Integer numTeams;
     private Integer mapSize;
 
-    @CommandLine.Option(names = "--map-size", order = 2, defaultValue = "512", description = "Generated map size, can be specified in oGrids (e.g 512) or km (e.g 10km)", showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
+    @CommandLine.Option(names = "--map-size", order = 4, defaultValue = "512", description = "Generated map size, can be specified in oGrids (e.g 512) or km (e.g 10km)", showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
```

Comment:
why did the order values change? Order is relative to the argGroup

Reply:
To be honest I didn't understand how the order of groups and options relates. I got the weird behavior that the heading of a group was at the very bottom while its contents were higher up. I didn't even think that was possible. So I fiddled with the values until at least the group headings were not detached from their content

Reply:
I reorganized the values and headings